File: go-client-sdk/pkg/models/shared/webhookmetadata.go
package shared

type WebhookMetadata struct {
	ConnectionID *string `json:"connectionId,omitempty"`
	CreatedTime  *string `json:"createdTime,omitempty"`
	WebhookID    *string `json:"webhookId,omitempty"`
	WebhookURL   *string `json:"webhookUrl,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/webhookmetadata.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class WebhookMetadata extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=connectionId" })
  connectionId?: string;

  @SpeakeasyMetadata({ data: "json, name=createdTime" })
  createdTime?: string;

  @SpeakeasyMetadata({ data: "json, name=webhookId" })
  webhookId?: string;

  @SpeakeasyMetadata({ data: "json, name=webhookUrl" })
  webhookUrl?: string;
}
